File: CS/ChangeDatabase.Web/Global_.asax.cs
using System;
using System.Collections.Generic;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using DevExpress.ExpressApp.Web;
using DevExpress.ExpressApp.Security;
using DevExpress.ExpressApp;
using System.Configuration;
using ChangeDatabase.Module;
using DevExpress.Persistent.BaseImpl;
using ChangeDatabase.Module.Web;

namespace ChangeDatabase.Web {
    public partial class Global : System.Web.HttpApplication {
        protected void Session_Start(Object sender, EventArgs e) {
            WebApplication.SetInstance(Session, new ChangeDatabaseAspNetApplication());

            ((SecurityBase)WebApplication.Instance.Security).Authentication = new AuthenticationStandard<SimpleUser, ChangeDatabaseStandardAuthenticationLogonParameters>();

            //((SecurityBase)WebApplication.Instance.Security).Authentication = new WebChangeDatabaseAuthenticationActiveDirectory();

            WebApplication.Instance.CanAutomaticallyLogonWithStoredLogonParameters = true;

            if(ConfigurationManager.ConnectionStrings["ConnectionString"] != null) {
                WebApplication.Instance.ConnectionString = ConfigurationManager.ConnectionStrings["ConnectionString"].ConnectionString;
            }
            WebApplication.Instance.Setup();
            WebApplication.Instance.Start();
        }
    }
}

File: VB/ChangeDatabase.Web/Global_.asax.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports DevExpress.ExpressApp.Web
Imports DevExpress.ExpressApp.Security
Imports DevExpress.ExpressApp
Imports System.Configuration
Imports ChangeDatabase.Module
Imports ChangeDatabase.Module.Web
Imports DevExpress.Persistent.BaseImpl

Namespace ChangeDatabase.Web
	Partial Public Class [Global]
		Inherits System.Web.HttpApplication
		Protected Sub Session_Start(ByVal sender As Object, ByVal e As EventArgs)
			WebApplication.SetInstance(Session, New ChangeDatabaseAspNetApplication())

			CType(WebApplication.Instance.Security, SecurityBase).Authentication = New AuthenticationStandard(Of SimpleUser, ChangeDatabaseStandardAuthenticationLogonParameters)()

			'CType(WebApplication.Instance.Security, SecurityBase).Authentication = New WebChangeDatabaseAuthenticationActiveDirectory()

			WebApplication.Instance.CanAutomaticallyLogonWithStoredLogonParameters = True

			If ConfigurationManager.ConnectionStrings("ConnectionString") IsNot Nothing Then
				WebApplication.Instance.ConnectionString = ConfigurationManager.ConnectionStrings("ConnectionString").ConnectionString
			End If
			WebApplication.Instance.Setup()
			WebApplication.Instance.Start()
		End Sub
	End Class
End Namespace